device/transport/amqp/src/amqp_twin_client.ts

```diff
@@ -161,7 +161,10 @@ export class AmqpTwinClient extends EventEmitter {
                   } else {
                     this._senderLink = senderTransportObject;
                     this._senderLink.on('error', this._errorHandler);
-                    this._client.attachReceiverLink( this._endpoint, this._generateTwinLinkProperties(linkCorrelationId), (receiverLinkError?: Error, receiverTransportObject?: any): void => {
+                    const linkOptions = this._generateTwinLinkProperties(linkCorrelationId);
+                    const autoAcceptPropertyName = 'autoaccept';
+                    linkOptions[autoAcceptPropertyName] = true;
```

Comment:
Is there an absolute need to manipulate this object or can we adjust this when it's created.

Reply:
Fixed.  Thanks for calling me out on that.